src/route-metadata.js

```diff
@@ -55,7 +56,7 @@ const preload = (content, resourcePath, skyPagesConfig) => {
         };
 
         if (preferredName !== undefined) {
-          routeMetadata.name = preferredName;
+          routeMetadata.name = jsonDataUtil.parseAngularBindings(preferredName);
```

Comment:
No need to parse this value again, as it was done already in `json-data-element-attributes`. :)

https://github.com/blackbaud/skyux-builder-plugin-stache/pull/15/files#diff-f365ee3414ecefcdac491266aa5ee256R7